mac/Keyman4MacIM/Keyman4MacIM/KMInputMethodEventHandler.m

```diff
@@ -489,13 +489,7 @@ - (BOOL)handleEvent:(NSEvent *)event client:(id)sender {
     /*if (event.type == NSKeyDown)
      [self.AppDelegate handleKeyEvent:event];*/
     
-    if (event.type == NSLeftMouseDown || event.type == NSLeftMouseUp ) {
-        _contextOutOfDate = YES;
```

Comment:
So is context out of date managed elsewhere now with mouse clicks?

Reply:
Yes, the low-level event tap sets the contextChangingEventDetected flag on the app delegate indicating that something has happened (mouse click or command-key) that has the potential to change our location. Then at line 295 of KMInputMethodEventHandler.m, we check that flag and set _contextOutOfDate accordingly. When I added that code (back in February?), I just didn't realize that it made this code irrelevant.